instrumentation/spring/spring-webmvc-5.3/library/src/main/java/io/opentelemetry/instrumentation/spring/webmvc/v5_3/HttpRouteSupport.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.spring.webmvc.v5_3;
+
+import static java.util.Objects.requireNonNull;
+import static org.springframework.web.util.ServletRequestPathUtils.PATH_ATTRIBUTE;
+
+import io.opentelemetry.context.Context;
+import java.lang.ref.WeakReference;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicBoolean;
+import javax.annotation.Nullable;
+import javax.servlet.FilterConfig;
+import javax.servlet.http.HttpServletRequest;
+import org.springframework.context.ApplicationListener;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.web.context.ConfigurableWebApplicationContext;
+import org.springframework.web.context.WebApplicationContext;
+import org.springframework.web.context.support.WebApplicationContextUtils;
+import org.springframework.web.servlet.DispatcherServlet;
+import org.springframework.web.servlet.HandlerExecutionChain;
+import org.springframework.web.servlet.HandlerMapping;
+import org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping;
+import org.springframework.web.util.ServletRequestPathUtils;
+
+final class HttpRouteSupport {
+
+  private final AtomicBoolean contextRefreshTriggerred = new AtomicBoolean();
```

Comment:
```suggestion
  private final AtomicBoolean contextRefreshTriggered = new AtomicBoolean();
```

Reply:
The intricacies of the English language... 🙈 
(On a side note, I wonder why IntelliJ did not highlight this)